src/plugins/validate-semantic/index.js

```diff
@@ -1,6 +1,6 @@
 import * as selectors from "./selectors"
 import * as actions from "./actions"
-import traverse from "traverse"
+import traverse from "neotraverse"
```

Comment:
May or may not need to be turned into neotraverse/legacy, whether it works or not